test/test_files/tinysnb/agg/multi_label.test

```diff
@@ -1,33 +1,9 @@
--NAME MultiLabelAggTest
--QUERY MATCH (a:person)-[e:marries|:mixed]->(b:person)-[:knows|:mixed]->(c:person) WHERE b.ID >= 7 RETURN b.ID, COUNT(*), SUM(a.ID + c.ID)
----- 2
-7|3|29
-8|1|10
+-NAME MultiLabelAggTest1
+-QUERY MATCH (a:person)<-[e1:marries|studyAt]-(b:person)-[e2:knows]->(c:person) WHERE b.ID = 7 RETURN ID(e1), COUNT(*), MIN(e2.date)
+---- 1
+7:2|2|1905-12-12
 
 -NAME MultiLabelAggTest2
--QUERY MATCH (a:person)<-[e1:marries|:mixed|studyAt]-(b:person)-[e2:knows]->(c:person) WHERE b.ID = 7 RETURN ID(e1), COUNT(*), MIN(e2.date)
----- 2
-7:3|2|1905-12-12
-8:2|2|1905-12-12
-
--NAME MultiLabelAggTest3
--QUERY MATCH (a:person)-[e1:marries|:mixed|studyAt]->(b:person) RETURN e1.year, COUNT(*)
```

Comment:
I think mixed can be rewrite as workAt|meets except there is one property missing. Can u rewrite multi-label test that are deleted due to `mixed`. We are less confidence on multi-labeled feature at this point.

Reply:
If i removed the mixed table, e.g. the MultiLabelAggTest3 has no difference between MultiLabelAggTest2. I rewrote some multi-labeled tests which are worthing keep.